.github/workflows/build&deploy-dev.yml

```diff
@@ -0,0 +1,120 @@
+name: Shipping to AWS ECS from Source Code
+  
+on:
+  workflow_dispatch:
+  push:
+    branches: [ "main" ]
```

Comment:
is deploying when push to main branch is unnecessary

Reply:
updated.